_posts/2021-02-05-homebrew-3.0.0.md

```diff
@@ -0,0 +1,34 @@
+- [The new `HOMEBREW_BOOTSNAP` environment variable allows the use of the Bootsnap gem to speed up repeated `brew` calls. This does not work (yet) on Apple Silicon or using Homebrew's portable Ruby](https://github.com/Homebrew/brew/pull/10374).
+- [Various methods have been deprecated, disabled and removed](https://github.com/Homebrew/brew/pull/10397)
+- [Bash](https://github.com/Homebrew/brew/pull/10229), [fish](https://github.com/Homebrew/brew/pull/10431) and [zsh](https://github.com/Homebrew/brew/pull/10403) completions are generated automatically from the `CLI::Parser` DSL. This will ensure they are kept up-to-date.
+- [`brew update` better handles upstream branch renames (e.g. from `master` to `main`)](https://github.com/Homebrew/brew/pull/10423)
+- [`brew bottle` and `bottle do` blocks use a new syntax format (one `:cellar` per platform)](https://github.com/Homebrew/brew/pull/10449). [`brew style --fix` will autocorrect formulae to this new format](https://github.com/Homebrew/brew/pull/10453).
+- [`brew completions` is a new command to opt-in to completions provided by third-party taps](https://github.com/Homebrew/brew/pull/10268)
+
+Other changes since 2.7.0 I'd like to highlight are the following:
+
+- [Command usage text is automatically generated (so will be kept-up-to-date)](https://github.com/Homebrew/brew/pull/10342)
+- [`brew audit` reads more formula data from taps](https://github.com/Homebrew/brew/pull/10087)
+- [`brew untap` of an official tap you don't use (e.g. Homebrew/homebrew-cask) will ensure it's no longer automatically retapped](https://github.com/Homebrew/brew/pull/10366)
+- [`brew casks` is a new command implemented in Bash to speedily output all casks available to install (like `brew formulae`)](https://github.com/Homebrew/brew/pull/10259)
+- [`brew info --cask --json=v2` includes whether a cask is outdated and the currently installed versions](https://github.com/Homebrew/brew/pull/10278)
+- [Fixed a bug where `brew update` could be run every time](https://github.com/Homebrew/brew/pull/10388)
+- [`brew --prefix --installed` is a new flag to `brew --prefix` that will fail if the requested formula is not installed](https://github.com/Homebrew/brew/pull/10266)
+- [We now use an unversioned SDK path on Big Sur to avoid breakage on minor SDK version changes](https://github.com/Homebrew/brew/pull/10072)
+
+Finally:
+
+- [Discourse](https://github.com/Homebrew/brew/pull/9144) was made read-only on January 1st 2021 in favour of [GitHub Discussions](https://github.com/Homebrew/brew/pull/8802).
```

Comment:
Shouldn't these link to the respective forums?

Reply:
fixed in latest push.